index.html

```diff
@@ -3,8 +3,9 @@
     <head>
         <meta charset="utf-8">
         <title>Javascript 101</title>
-        <script src="scripts/oshomo.oforomeh-v2.js"></script>
+        <script src="scripts/scripts/oshomo.oforomeh-v2.js"></script>
```

Comment:
You have script twice. it should be 
```suggestion
        <script src="scripts/oshomo.oforomeh-v2.js"></script>
```